config/packages/doctrine.yaml

```diff
@@ -18,7 +18,91 @@ doctrine:
         mappings:
             App:
                 is_bundle: false
-                type: annotation
-                dir: '%kernel.project_dir%/src/Entity'
-                prefix: 'App\Entity'
+                type: yml
```

Comment:
Can we stay with one type of mapping among all files by default? I prefer to use XML, but even more, I prefer to remain consistent.

Reply:
Also, maybe we should migrate to another configuration form and recommend it to all projects?

Reply:
The thing is, we need to keep annotation mapping for Sylius classes as they can be extended in Sylius plugins and it can be _properly_ done only by traits with annotation mapping... But I agree with keeping one type of mapping by default 👍